js/index.js

```diff
@@ -17,3 +17,125 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 }
+
+
+const messageForm = document.querySelector('form[name="leave_message"]');
+const messageSection = document.querySelector('#messages');
+const messageList = messageSection.querySelector('ul');
+
+messageSection.hide = function () {
+    this.style.display = 'none';
+};
+messageSection.show = function () {
+    this.style.display = '';
+};
+
+Object.defineProperty(messageList, "hasNoMessages", {
+    get: function () {
+        return this.childElementCount === 0
+    }
+});
+
+function updateMessageSection() {
+    if (messageList.hasNoMessages) {
+        messageSection.hide();
+    }
+};
+updateMessageSection();
+
+
+messageForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const form = e.target;
+
+    const userName = form.usersName.value;
+    const useEmail = form.usersEmail.value;
+    const userMessage = form.usersMessage.value;
+
+    // console.log(userName);
+    // console.log(useEmail);
+    // console.log(userMessage);
+
+    const newMessage = document.createElement('li');
+    newMessage.innerHTML = `
+        <a href="mailto:${useEmail}" target="_blank">${userName}</a>
+        wrote: <span> ${userMessage}</span>
+    `;
+    newMessage.messageText = userMessage;
+
+    const removeButton = document.createElement('button');
+    removeButton.innerText = "remove";
+    removeButton.type = "button";
+
+    const editButton = document.createElement('button');
+    editButton.innerText = "edit";
+    editButton.type = "button";
+
+
+    newMessage.appendChild(editButton);
+    newMessage.appendChild(removeButton);
+    messageList.appendChild(newMessage);
+    messageSection.show();
+
+    form.reset();
+});
+
+messageList.addEventListener('click', (e) => {
+    if (e.target.tagName === 'BUTTON') {
+        const button = e.target;
+        const listButtons = messageList.querySelectorAll('button');
+        const li = button.parentNode;
+        const messageText = li.messageText;
+        const action = button.textContent;
+
+        const disableButton = (btn) => {
+            if (btn.parentNode !== li) {
+                btn.disabled = true;
+            }
```

Comment:
I thought about one thing: if someone left a message in the form "Leave a Message" and then wanted to edit this message.
So below there will be all messages are available to edit. 
I am user, I click "edit" button, and at the same time while I am editing the message, I can click edit button on the other message.
So I can't think of anything better than to disable all other buttons (for now and to change the code later, when I understand what to do with this).
Or I shoul add one more parameter like this?
`        const disableButton = (button, parent) => {
            if (button.parentNode !== parent) {
                button.disabled = true;
            }
        };`

Reply:
Ah ok I see what you mean! Yeah this kind of edgecase is interesting. It might not specifically cause a problem in this specific app if someone is editing two messages at the same time, but I agree that might not be the UX you want. 

Ok I say leave this as-is for now, and when you get into React, you'll be able to use State to keep track of things like which elements are currently being edited!